satellite/repair/checker/checker.go

```diff
@@ -55,6 +56,7 @@ type Checker struct {
 	metainfo        *metainfo.Service
 	metaLoop        *metainfo.Loop
 	nodestate       *ReliabilityCache
+	repairOverride  int32
```

Comment:
Just curious why are we using `int32` here?

Reply:
Our RS parameters are int32 (probably due to the protobuf definition). To make comparison easier and less error prone, I have tried to keep it to int32 where possible.